cmake/linux/default_libs.cmake

```
@@ -10,15 +10,21 @@ execute_process (COMMAND
     COMMAND_ERROR_IS_FATAL ANY
     OUTPUT_STRIP_TRAILING_WHITESPACE)
 
+if (NOT EXISTS "${BUILTINS_LIBRARY}")
+    include (cmake/build_clang_builtin.cmake)
+    if (CMAKE_CXX_COMPILER_TARGET STREQUAL "x86_64-linux-musl")
+        build_clang_builtin("x86_64-linux-musl" BUILTINS_LIBRARY)
+    else ()
+        message (WARNING "Builtins library not found at ${BUILTINS_LIBRARY} and no build rule for target ${CMAKE_CXX_COMPILER_TARGET}. Using libgcc as fallback.")
+        set (BUILTINS_LIBRARY "-lgcc")
+    endif ()
```

Maybe this will be better? (And update the function accordingly)

```suggestion
    build_clang_builtin("x86_64-linux-musl" BUILTINS_LIBRARY)
    if (NOT BUILTINS_LIBRARY)
        message (WARNING "Builtins library not found at ${BUILTINS_LIBRARY} and no build rule for target ${CMAKE_CXX_COMPILER_TARGET}. Using libgcc as fallback.")
        set (BUILTINS_LIBRARY "-lgcc")
    endif ()
```